CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -198,11 +198,58 @@ def __init__(self, db_client: 'botocore.client.DynamoDB'):
         super().__init__(db_client=db_client)
 
 class FridgeHistory(DB_Item):
-    
+
     def __init__(self, db_client: 'botocore.client.DynamoDB'):
         super().__init__(db_client=db_client)
 
-class Tag(DB_Item):
 
-    def __init__(self, db_client: 'botocore.client.DynamoDB'):
+class Tag(DB_Item):
+    REQUIRED_FIELDS = ['tag_name']
+    TABLE_NAME = "tag"
+    def __init__(self, db_client: 'botocore.client.DynamoDB', tag_name:str= None):
         super().__init__(db_client=db_client)
+        self.tag_name = tag_name
+
+    def format_tag(self, tag_name:str) -> str:
```

Comment:
We never actually use this function? 

And I think this function might be doing too much?

```
def format_tag(self, tag_name:str) -> str:
    if tag_name:
        tag_name = tag_name.lower().replace(" ", "")
    return tag_name
```

And you can then call this function within the constructor

```
self.tag_name = format_tag(tag_name)
```